### PR TITLE
Adding Swoole's Server mode and settings

### LINKED ISSUE #58: [Swoole] Support Static File Handler
I just try to use runtime using swoole runner but i got 404 when serving static files

`No route found for "GET http://localhost:9876/favicon.ico"`

Basically i use `https://github.com/k911/swoole-bundle` and have no issue about it and want to try to use symfony runtime

### PR DESCRIPTION
**Pull request by leocavalcante**
Closes #58 by
```php
$_SERVER['APP_RUNTIME_OPTIONS'] = [
    'settings' => [
        \Swoole\Constant::OPTION_ENABLE_STATIC_HANDLER => true,
        \Swoole\Constant::OPTION_DOCUMENT_ROOT => dirname(__DIR__).'/public'
    ]
];
```

### CODE REVIEW

**Comment by ad3n** [comment]
Screenshot from 2021-06-22 10-03-53

It's same... I put that in `index.php`

```php
$_SERVER['APP_RUNTIME_OPTIONS'] = [
    'settings' => [
        Constant::OPTION_ENABLE_STATIC_HANDLER => true,
        Constant::OPTION_DOCUMENT_ROOT => '/semart/public',
    ]
];
```

**Comment by leocavalcante** [comment]
@ad3n this isn't merged yet.

**Review by Nyholm — APPROVED** [review]
Lovely. This is great. 

Could you add a small test to it?

**Comment by leocavalcante** [comment]
Sure, I will think on something this weekend.

**Comment by leocavalcante** [comment]
@Nyholm I think this is it.
I have a `phpunit.xml` with
```xml
<testsuite name="E2E">
  <directory>./tests/E2E</directory>
</testsuite>
```
added.

I wasn't sure how to add a working server to the CI for the E2E, but Unit tests covers 100%.

File: src/swoole/composer.json [hunk 1]
```diff
@@ -13,6 +13,8 @@
         "symfony/runtime": "^5.3 || ^6.0"
     },
     "require-dev": {
+        "ergebnis/composer-normalize": "^2.15",
```
Comment (Nyholm) [inline src/swoole/composer.json:16]: This is not required here

**Review by Nyholm — COMMENTED** [review]
Isn't all test suites running by default? Ie, we are already running E2E on the CI?

**Comment by leocavalcante** [comment]
Thanks!

All test suites will run, but on the `.dist` PHPUnit there is only the `./tests/Unit`.
I didn't saw a step on the CI for spinning up a runtime and executing E2E, it's kind of a manual step right now:
```shell
php tests/E2E/runtime.php
```
Then:
```shell
phpunit --testsuit E2E
```

**Comment by Nyholm** [comment]
Great. Thank you. Let's improve CI later.

**Pull request merged**